plugin.rb

```diff
@@ -270,18 +235,18 @@ def voted_by
 
     require_dependency 'topic_query'
     class ::TopicQuery
-      SORTABLE_MAPPING["votes"] = "custom_fields.#{::DiscourseVoting::VOTE_COUNT}"
-
       def list_voted_by(user)
         create_list(:user_topics) do |topics|
-          topics.where(id: user.custom_fields[DiscourseVoting::VOTES])
+          topics
+            .joins("LEFT JOIN discourse_voting_votes ON discourse_voting_votes.topic_id = topics.id")
```

Comment:
Should this be an inner join instead since we only want the intersection between `topics` and `discourse_voting_votes`.